guests_app/src/ViewController/MainPageViewController.tsx

```diff
@@ -1,23 +1,25 @@
 import Requests from 'guests_app/src/Networking/requests';
-import React from 'react';
+import React, {useContext, useState} from 'react';
 import {Alert, PermissionsAndroid, Platform} from 'react-native';
+import {itemsContext, MyLocationContext, OrdersContext} from '../contexts';
+import {OrderID} from '../types';
 import {MainPage} from '../View/MainPageView';
-import {ItemsViewModel} from '../ViewModel/ItemViewModel';
+import ItemViewModel from '../ViewModel/ItemViewModel';
 import OrderViewModel from '../ViewModel/OrderViewModel';
 //import {observer} from 'mobx-react-lite';
 
 export const MainPageViewController = () => {
-	/**
-	 @todo: initialize all View Models at start
-	**/
-	const requests = new Requests();
-	const orderViewModel = new OrderViewModel(requests);
-	const itemViewModel = new ItemsViewModel(requests);
-	itemViewModel.syncItems();
+	const orderViewModel = useContext(OrdersContext);
+	const itemViewModel = useContext(itemsContext);
+	const locationViewModel = useContext(MyLocationContext);
 
-	//	const [waitingForOrder, setWaitingForOrder] = useState(false);
-	//	const order_items = ['bamba', 'Beer'];
-	//	const [orderID, setOrderID] = useState('');
+	const [orderID, setOrderID] = useState('');
```

Comment:
Should'nt the varibles be stored in a model? id, status and active.
The server can change these through the websockets and you wont be notified in the component